frontends/aiger/aigerparse.cc

```diff
@@ -131,36 +269,199 @@ void AigerReader::parse_aiger()
 		std::getline(f, line); // Ignore up to start of next line
 	}
 
-	module->fixup_ports();
-	design->add(module);
+	post_process();
+}
+
+static uint32_t parse_xaiger_literal(std::istream &f)
+{
+	uint32_t l;
+	f.read(reinterpret_cast<char*>(&l), sizeof(l));
+	if (f.gcount() != sizeof(l))
+		log_error("Offset %ld: unable to read literal!\n", static_cast<int64_t>(f.tellg()));
```

Comment:
I think this won't work on a typical 32-bit system which has 32-bit longs (so `%ld` will be expecting a 32-bit value)